### PR TITLE
fix #105

### LINKED ISSUE #105: [feature] Hide cpuspeed from UI and only show CPU number
From ZStack 1.3, CPU speed option (Instance Offering) will not be directly used. We will use CPU over provision rate to control how many vCPUs could be created on physical CPUs. 

For example, if the host has 8 physical CPU and CPU over provision rate is 10, it means we can totally create VMs with 80 vCPUs.

The CPU over provision rate can be set through global configuration and get by QueryGlobalConfiguration API. For example:

```
# zstack-cli QueryGlobalConfig category=host name=cpu.overProvisioning.ratio
{
    "inventories": [
        {
            "category": "host",
            "defaultValue": "10",
            "description": "The over provisioning ratio for CPU number",
            "name": "cpu.overProvisioning.ratio",
            "value": "10"
        }
    ],
    "success": true
}
```

With this new feature, we should change zstack-dashboard:
1, In Instance Offering, we need to hide "CPU Speed" to user and always set it to 1 when calling CreateInstanceOffering API.
2, In dashboard portal, we don't need to show CPU frequency usage, but to use vCPU usage instead. The value should be "current vCPUs number"/(Host CPU number * CPU over provision rate)

### PR DESCRIPTION
**Pull request by tolbkni**

### CODE REVIEW

**Comment by tolbkni** [comment]
@youyk help to test

File: ts/utils.ts [hunk 1]
```diff
@@ -659,8 +659,8 @@ module Utils {
         }
     }
 
-    export function toHZString (input) {
-        return input + ' HZ';
+    export function toVCPUString (input) {
+        return input + ' VCPUs';
```
Comment (tolbkni) [inline ts/utils.ts]: I add VCPUs here to describe the value clear.

**Comment by youyk** [comment]
Looks good. Let me merge it firstly and test it.

**Comment by youyk** [comment]
I have tested it. Very good.

**Pull request merged**